packages/website/pages/terms.js

```diff
@@ -23,75 +23,158 @@ export default function TermsOfService() {
         <p className="lh-copy">
           The following terms and conditions govern all use of the{' '}
           <a className="black" href="https://nft.storage">
-            nft.storage
+            NFT.Storage
           </a>{' '}
           website (the “Website”) and all content, services and products
```

Comment:
- suggestion: "...and all content, services, and products" (consistent usage of oxford comma)